src/libfetchers/git.cc

```diff
@@ -148,8 +148,14 @@ std::vector<PublicKey> getPublicKeys(const Attrs & attrs)
     std::vector<PublicKey> publicKeys;
     if (attrs.contains("publicKeys")) {
         nlohmann::json publicKeysJson = nlohmann::json::parse(getStrAttr(attrs, "publicKeys"));
-        ensureType(publicKeysJson, nlohmann::json::value_t::array);
-        publicKeys = publicKeysJson.get<std::vector<PublicKey>>();
+        auto pubKeys = getArray(publicKeysJson);
```

Comment:
```suggestion
        auto & pubKeys = getArray(publicKeysJson);
```
If the function returns an array and you don't want to copy the result, you have to use `auto &` not `auto`.